Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -328,7 +328,7 @@ import BraintreeDataCollector
                 case ASWebAuthenticationSessionError.canceledLogin:
                     // User canceled by breaking out of the PayPal browser switch flow
                     // (e.g. System "Cancel" button on permission alert or browser during ASWebAuthenticationSession)
-                    if self.returnedToAppAfterPermissionAlert == true {
+                    if self.returnedToAppAfterPermissionAlert == false {
```

Comment:
I had a misunderstanding of this variable. It means that it had returned from launch of the webView, so true would mean it was canceled from browser.

Reply:
Why is this changing from true to false?

Reply:
I believe this is working as expected on `master`

Reply:
I tested this several times.

Reply:
Ok I see - The true case ([used in PayPal](https://github.com/braintree/braintree_ios/blob/495738201776c5ce79d907c4ac9511a5120e0a05/Sources/BraintreePayPal/BTPayPalClient.swift#L327)) means the cancel button on the web view was clicked.

Reply:
Ok I see, so this is fixing a mistake introduced in[ this PR](https://github.com/braintree/braintree_ios/pull/977)